platform-includes/sourcemaps/upload/primer/javascript.sveltekit.mdx

```diff
@@ -0,0 +1,82 @@
+### Source Maps Upload
+
+By default, `sentrySvelteKit()` will add an instance of the Sentry Vite Plugin, to upload source maps for both server and client builds. This means that when you run a production build (`npm run build`), source maps will be generated and uploaded to Sentry so that you get readable stack traces in your Sentry issues.
```

Comment:
l: I'd reword this slightly to better explain `sentrySvelteKit()`. I think in the previous location, the meaning was implicitly clear from the sections above it but on this page it needs further clarification. Also, we can just omit the Vite plugin instance because at the end of the day that's an implementation detail. 

```suggestion
By default, the `sentrySvelteKit()` Vite plugin uploads source maps for both server and client builds. This means that when you run a production build (`npm run build`), source maps will be generated and uploaded to Sentry so that you get readable stack traces in your Sentry issues.
```